dev-libs/mtxclient/mtxclient-0.7.0.recipe

```diff
@@ -25,8 +25,12 @@ REQUIRES="
 	lib:libboost_iostreams$secondaryArchSuffix
 	lib:libboost_system$secondaryArchSuffix
 	lib:libboost_thread$secondaryArchSuffix
```

Comment:
Why is boost still part of the dependencies?

Reply:
Was checking the source on github when I saw you removed it from the dependencies, should indeed be removed.